src/fontstash.h

```diff
@@ -811,6 +858,8 @@ void fonsClearState(FONScontext* stash)
 	state->font = 0;
 	state->blur = 0;
 	state->spacing = 0;
+	state->oversampleH = 2;
```

Comment:
Also in #266 you actually mentioned 2x2 oversample would be a better default... still waiting to get back to you on the blur issue though.